desktop/apps/auction/components/filter_sort/index.js

```diff
@@ -42,6 +44,12 @@ function FilterSort(props) {
   )
 }
 
+FilterSort.propTypes = {
```

Comment:
I think thats where things are headed, but of course for a large discussion. `standard-react` screams without them, which in the long term is probably more of a good thing than bad as data dependencies change over time.